src/alejandra/src/builder.rs

```diff
@@ -99,7 +100,12 @@ fn build_step(
                 add_token(
                     builder,
                     rnix::SyntaxKind::TOKEN_WHITESPACE,
-                    &format!("{0:<1$}", "", 2 * build_ctx.indentation),
+                    &match build_ctx.config.indentation {
```

Comment:
Allowing customization with an unlimited option space may have disadvantages as well, for instance, it would be very atypical for someone to select 2 tabs as indentation, and very odd to read code indented at 13 or 999 spaces

I'm not opposed to supporting other options down the road (for instance 8 spaces if somebody wants that), but I would prefer the tool to provide a sensible option space where users can't go wrong with any option

This is all still subject to change as feedback comes